docs/cookbooks/common_workflows.md

```diff
@@ -91,16 +91,7 @@ Adding a new field to a table schema also means that the field has to propagate
 
 1. Open the `query.sql` file inside the `<dataset>.<table>` location and add the new definitions for the field.
 1. Run `./bqetl format <path to the query>` to format the query. Alternatively, run `./bqetl format $(git ls-tree -d HEAD --name-only)` validate the format of all queries that have been modified.
-1. Run `./bqetl query validate <dataset>.<table>` to dry run the query.
-
-   For data scientists (and anyone without `jobs.create` permissions in `moz-fx-data-shared-prod`), run: 
-   
-   ```
-    gcloud auth login --update-adc   # to authenticate to GCP
-    gcloud config set project mozdata    # to set the project
-    ./bqetl query validate --use-cloud-function=false --project-id=mozdata <full path to the query file>
-   ```
-
+1. Run `./bqetl query validate <dataset>.<table>` to dry run the query. For data scientists (and anyone without `jobs.create` permissions in `moz-fx-data-shared-prod`), instead run: (a) `gcloud auth login --update-adc  # to authenticate to GCP`  (b) `gcloud config set project mozdata    # to set the project`  (c) `./bqetl query validate --use-cloud-function=false --project-id=mozdata <full path to the query file>`
```

Comment:
I think the blank lines break the formatting here, but this should work:
```suggestion
1. Run `./bqetl query validate <dataset>.<table>` to dry run the query.
    * For data scientists (and anyone without `jobs.create` permissions in `moz-fx-data-shared-prod`), run:
    ```
    gcloud auth login --update-adc   # to authenticate to GCP
    gcloud config set project mozdata    # to set the project
    ./bqetl query validate --use-cloud-function=false --project-id=mozdata <full path to the query file>
   ```
```

Reply:
thanks!